st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
+                "type": "array",
+                "items": {"type": "string"},
+                "description": "Parameter format hidden from public list."
+            },
+            "ack": {
```

Comment:
Do we ever want to support any other ack related attributes? If not, it's probably better to just use a simple "boolean" attribute (e.g. `require_ack`).